src/Uno.UI/UI/Xaml/Duration.cs

```diff
@@ -28,8 +28,7 @@ public bool HasTimeSpan
 		{
 			get
 			{
-				return this.Type == DurationType.TimeSpan &&
-					this.TimeSpan.CompareTo(TimeSpan.Zero) > 0;
+				return this.Type == DurationType.TimeSpan;
```

Comment:
This modifies the public behavior, is this is valid?

Reply:
@dr1rrb That appears to match Windows from my testing.